modules/auxiliary/scanner/ipmi/ipmi_dumphashes.rb

```diff
@@ -180,18 +179,8 @@ def run_host(ip)
       write_output_files(rhost, username, sha1_salt, sha1_hash)
 
       # Write the rakp hash to the database
-      report_auth_info(
-        :host	=> rhost,
-        :port   => rport,
-        :proto  => 'udp',
-        :sname	=> 'ipmi',
-        :user 	=> username,
-        :pass   => "#{sha1_salt}:#{sha1_hash}",
-        :source_type => "captured",
-        :active => true,
-        :type   => 'rakp_hmac_sha1_hash'
-      )
-
+      hash = "#{rhost} #{username}:$rakp$#{sha1_salt}$#{sha1_hash}"
```

Comment:
I copied the hash format from the `write_output_files` method in this same module, but I didn't try by myself which the format is correct for john the ripper. I guess it is. Maybe worths from @hmoore-r7 / @dmaloney-r7 since they are the ipmi / jhon the ripper experts :)